web/docs/tql2/operators/fluentbit.md

```diff
@@ -15,27 +15,14 @@ and process events with a Fluent Bit [output plugin][outputs].
 [inputs]: https://docs.fluentbit.io/manual/pipeline/inputs
 [outputs]: https://docs.fluentbit.io/manual/pipeline/outputs
 
-Syntactically, the `fluentbit` operator behaves similar to an invocation of the
-`fluent-bit` command line utility. For example, the invocation
-
-```bash
-fluent-bit -o plugin -p key1=value1 -p key2=value2 -p ...
-```
-
-translates to our `fluent-bit` operator as follows:
-
-```tql
-fluentbit "plugin", ...
-```
-
```

Comment:
I think this is good reference content and should stay in the reference.

Reply:
We have it in the integration page now. This is the first entry point for a user who learns about how to use Tenzir and Fluent Bit. Therefore, it belongs there.